### PR TITLE
Add info about installing packs from private repo

### PR DESCRIPTION
**Pull request by lakshmi-kannan**

### CODE REVIEW

File: docs/source/packs.rst [hunk 1]
```diff
@@ -51,6 +51,12 @@ By default packs are installed from the |st2| community repo. Use ``repo_url`` p
 
     st2 run packs.install register=all repo_url=https://github.com/StackStorm/st2incubator.git
 
+If you are using a private repo to host your pack(s),
+``repo_url=https://username:password@github.com/username/repository.git`` should get you going.
```
Comment (manasdk) [inline docs/source/packs.rst]: Is this github specific? I don't see it in https://www.kernel.org/pub/software/scm/git/docs/git-clone.html#URLS

Even if it is github specific it should be fine to include so long as we call it out.

Comment (lakshmi-kannan) [inline docs/source/packs.rst]: Yep, github specific. Let me add that to the explanation.

Comment (lakshmi-kannan) [inline docs/source/packs.rst]: Fixed in bfd4c28

**Comment by manasdk** [comment]
+1

**Pull request merged**